### ISSUE TITLE
Tomcat fails to instantiate the plugin

### ISSUE THREAD

**Issue by SFAntti** [camunda/camunda-bpm-reactor#45]
I'm trying to use the ReactorProcessEnginePlugin when running Camunda in Tomcat (using the default Camunda 7.5.0 Tomcat distribution).

However, instantiation of the plugin fails due to not having a constructor that takes no parameters. See the error callstack in the attachment.

I believe a simple fix would be to add that missing version of the constructor and can provide a patch for that.

[camunda-bpm-reactor-tomcat-callstack.txt](https://github.com/camunda/camunda-bpm-reactor/files/537256/camunda-bpm-reactor-tomcat-callstack.txt)

**Comment by jangalinski** [comment]
I guess the Plugin is registered in the platform.xml by its FQN? Then the missing default constructor is a problem in deed. Thanks for reporting. Earlier version had a default constructor that created the reactor-eventbus. I removed it to make generation of the plugin more explicit (no hidden dependencies). But I only worked with embedded engines and plain-java configuration since then.

But as you mentioned ... this will be an easy fix.

**Comment by SFAntti** [comment]
Yes, the plugin was registered in the bpm-platform.xml by its FQN.

Thanks @jangalinski for such a quick fix!

**Comment by jangalinski** [comment]
see #103 .... since we do not test this, it got broken again
